### PR TITLE
update readme.md

### PR DESCRIPTION
**Pull request by sgpritam**
Improve Readme.md  #8

### CODE REVIEW

**Comment by sgpritam** [comment]
@rajat2502 Please review

File: README.md [hunk 1]
```diff
@@ -1,72 +1,93 @@
...
+
+* This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+
+### Deployment
+
+* This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+
+### `npm run build` fails to minify
+
+* This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Comment (rajat2502) [inline README.md]: No need to include this

File: README.md [hunk 2]
```diff
@@ -1,72 +1,93 @@
...
+
+7. Builds the app for production to the `build` folder.<br>
+	It correctly bundles React in production mode and optimizes the build for the best performance.
+
+	The build is minified and the filenames include the hashes.<br>
+	Your app is ready to be deployed!
+
+		`npm run build`
+
+
```
Comment (rajat2502) [inline README.md]: Remove all extra spaces like this from the file

File: README.md [hunk 3]
```diff
@@ -1,72 +1,93 @@
...
+### Learn More
+
+* You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+* To learn React, check out the [React documentation](https://reactjs.org/).
+
+### Code Splitting
+
+* This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
+
```
Comment (rajat2502) [inline README.md]: No need of this section  as well

File: README.md [hunk 4]
```diff
@@ -1,72 +1,93 @@
...
+  * React Hooks
+  * GitHub API
+  * Netlify.com (for hosting)
+
+
+### How to Run Locally :
+
+1. Clone the Repository
+  
+     `git clone https://github.com/NJACKWinterOfCode/Ravenous`
```
Comment (rajat2502) [inline README.md]: All the commands should be on the next line to its point

File: README.md (hunk 2 above)
Comment (rajat2502) [inline README.md]: Remove this section

**Review by rajat2502 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sgpritam) [reply README.md]: @rajat2502 Please review

**Pull request merged**